odoo/tools/translate.py

```diff
@@ -472,7 +472,12 @@ def _get_translation(self, source, module=None):
                 if not module:
                     path = inspect.getfile(frame)
                     path_info = odoo.modules.get_resource_from_path(path)
-                    module = path_info[0] if path_info else None
+                    if path_info:
+                        module = path_info[0]
+                    elif path.startswith(config['root_path']):
+                        module = 'base'
+                    else:
+                        module = None
```

Comment:
Thanks for the contribution 😃
We can simply replace the original line by
`module = path_info[0] if path_info else 'base'`

Reply:
@HydrionBurst thank you for suggestion, please review again